classes/controller/AdminController.php

```diff
@@ -3661,7 +3661,7 @@ public function validateRules($class_name = false)
             if (isset($def['lang']) && $def['lang']) {
                 if (isset($def['required']) && $def['required']) {
                     $value = Tools::getValue($field.'_'.$default_language->id);
-                    if (empty($value)) {
+                    if (!isset($value) || "" == $value) {
```

Comment:
Having a comment for this line would have been great. I'm pretty sure anybody seeing this will think `empty($value)` is equivalent.